scripts/lint-packages.sh

```diff
@@ -118,15 +118,36 @@ check_indentation() {
 lint_package() {
 	local package_script
 	local package_name
+	local git_root
 
 	package_script="$1"
 	package_name="$(basename "$(dirname "$package_script")")"
+	git_root="$(git rev-parse --show-toplevel)"
 
 	echo "================================================================"
 	echo
 	echo "Package: $package_name"
 	echo
 
+	echo -n "Layout: "
+	local section in_dir=''
+	for section in 'packages' 'x11-packages' 'root-packages'; do
```

Comment:
It is better to read them from `repo.json`.

Reply:
That's the sort of stuff I don't think about at 4 in the morning.

Reply:
Should rename `section` to `channel`.

https://help.ubuntu.com/community/InstallingSoftware#Software_Channels.2FRepositories

Reply:
yep that makes sense.